ddtrace/tracer/util.go

```diff
@@ -13,6 +13,8 @@ import (
 // toFloat64 attempts to convert value into a float64. If it succeeds it returns
 // the value and true, otherwise 0 and false.
 func toFloat64(value interface{}) (f float64, ok bool) {
+	const max = (int64(1) << 53)
+	const min = (-max)
```

Comment:
Could we be more explicit here? Those are the `min` and `max` of what?
I do not have strong opinions about "longer const names" vs "comment explaining what that const is about" (or to be more precise I tend to prefer "longer const names" but I am not asking you to adhere to it). So please pick the approach that you feel like being the most idiomatic for this use case.

Reply:
Since they're only to be used in this function, I'll keep them here. I've added description to the comment to describe why the function is doing what it's doing.